crates/binjs_io/src/lib.rs

```diff
@@ -408,20 +408,30 @@ impl Format {
         &multipart::FormatProvider
     }
 
+    /// Returns command-line arguments.
```

Comment:
Nit: Could you mention that advanced flags are hidden behind a command `"advanced"`?

Reply:
Also, could you turn this `"advanced"` into a named constant?